mobileconnectsdk/Mobile Connect SDK 17.0 Release Notes.md

```diff
@@ -0,0 +1,84 @@
+# Mobile Connect SDK 17.0 Release Notes
+
+The SDK update includes small bug fixes and BLE improvements for both platforms.  
+
+The following are the specific changes for each platform:
+
+## iOS SDK Changes
+
+* The SDK added `Wallet` feature support, allowing the user to provision Apple Wallet passes for registered credentials.
+* The Wallet feature (Apple Employee Badge) is a licensed feature that requires Command Centre server version 9.10 or higher.
```

Comment:
* The Wallet feature (Apple Employee Badge) is a licensed feature that requires Command Centre server version 9.10 MR3 or higher.